makejdk-local-env.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Script to clone the OpenJDK source then build it
+
+# Optionally uses Docker, otherwise you can provide two arguments:
+# the area to build the JDK e.g. $HOME/mybuilddir as -s or --source
+# and the target destination for the tar.gz e.g. -d or --destination $HOME/mytargetdir
+# Both must be absolute paths! You can use $PWD/mytargetdir
+
+# A simple way to install dependencies persistently is to use our Ansible playbooks
+
+# You can set the JDK boot directory with the JDK_BOOT_DIR environment variable
+
+set -eu
+
+OS_KERNAL_NAME=$(echo $(uname) | awk '{print tolower($0)}')
```

Comment:
echo => sets the final outcome of the whole command `$(uname) | awk '{print tolower($0)}'` to `OS_KERNEL_NAME` - otherwise the output might get executed